ReactWindows/ReactNative/Views/Scroll/ReactScrollViewManager.cs

```diff
@@ -387,23 +410,52 @@ protected override void AddEventEmitters(ThemedReactContext reactContext, Scroll
         private void OnDirectManipulationCompleted(object sender, object e)
         {
             var scrollViewer = (ScrollViewer)sender;
+
             EmitScrollEvent(
                 scrollViewer,
                 ScrollEventType.EndDrag,
                 scrollViewer.HorizontalOffset,
                 scrollViewer.VerticalOffset,
                 scrollViewer.ZoomFactor);
+            
+            var pageWidth = scrollViewer.Width;
+
```

Comment:
Any reason to use ScrollView with `pagingEnabled` instead of the `FlipView` we offer in UWP? It's pretty trivial with React Native to customize components per platform.